src/hooks/useGetFallbackToUseByAsset.ts

```diff
@@ -0,0 +1,25 @@
+import { Address } from "wagmi"
+
+import { VaultType } from "@/lib/types"
+import { useActiveChainId } from "@/hooks"
+
+const FALLBACK_TO_USE_BY_ASSET: Record<number, Record<Address, VaultType>> = {
```

Comment:
Same here -- should we just have something on the config for this?

Reply:
Agreed